.circleci/config.yml

```diff
@@ -93,24 +93,24 @@ jobs:
           command: "echo $DOCKER_PASSWORD | docker login --username $DOCKER_USERNAME --password-stdin"
       - run:
           name:  "Build Backend Docker image"
-          command: "cd tdrs-backend; docker build -t goraftdocker/tdp-backend:$CIRCLE_BRANCH . -f ./docker/Dockerfile.dev"
```

Comment:
This shouldn't really be showing up as a diff since these changes were already merged in with https://github.com/HHS/TANF-app/pull/130, I think this has to do with the `Merge branch 'raft-tdp-main' into improve-readmes` commit.

Reply:
Not a blocker, just a GitHub oddity! QASP reviewing now.